### PR TITLE
added requirements.txt and updated readme.md

### PR DESCRIPTION
**Pull request by sparkingdark**
Added Requirement.txt for better visibility in cifar10 example.

Description:

I saw that cifar210 example has several pip install commands for libraries instead of doing that just added a requirement.txt to install single command.

tagging @vfdev-5 is it okay or need to update more?

Check list:
* [ ] New tests are added (if a new feature is added)
* [ ] New doc strings: description and/or example code are in RST format
* [x] Documentation is updated (if required)

### CODE REVIEW

File: examples/contrib/cifar10/README.md [hunk 1]
```diff
@@ -23,6 +23,8 @@ Configurations:
 - [python-fire](https://github.com/google/python-fire): `pip install fire`
 - Optional: [clearml](https://github.com/allegroai/clearml): `pip install clearml`
 
+Alternatively install the all requirements using `pip install -r requirements.txt` command instead of typing pip several times.
```
Comment (vfdev-5) [inline examples/contrib/cifar10/README.md:26]:
```suggestion
Alternatively, install the all requirements using `pip install -r requirements.txt`.
```

File: examples/contrib/cifar10/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+pytorch-ignite
+torchvision
```
Comment (vfdev-5) [inline examples/contrib/cifar10/requirements.txt:2]:
```suggestion
torchvision
tqdm
```

**Review by vfdev-5 — CHANGES_REQUESTED** [review]
Thanks for the PR @sparkingdark . Few things to update.

**Comment by sparkingdark** [comment]
Okay will do soon

**Comment by sparkingdark** [comment]
@vfdev-5 is it solved now the changes?

**Review by vfdev-5 — APPROVED** [review]
LGTM!

**Comment by sparkingdark** [comment]
Thanks, @vfdev-5 for this quick review, I am now looking into the library you mentioned for high-level API project.

**Pull request merged**